### ISSUE TITLE
Yoshi's Woolly World - graphic packs break map screen

### ISSUE THREAD

**Issue by daninthemix** [slashiee/cemu_graphic_packs#150]
Have tried both 1080p and 2160p packs, and they cause the map screen to become pitch black, making the game impossible to play. No issues when all packs are disabled. This is on Cemu 1.11.2

**Comment by Crementif** [comment]
Shader packs???? This is a graphic pack repository, not even closely related to shader caches.

**Comment by daninthemix** [comment]
I meant graphic packs - it was a misspelling. There IS an issue with these packs.

**Comment by Crementif** [comment]
Okay, reopening this issue. I'll assign this to myself to do later today hopefully.

**Comment by Crementif** [comment]
![Fixed Map Screen](https://user-images.githubusercontent.com/26669564/34278850-ab109928-e6ad-11e7-8baa-65778060ca7d.png) Fixed in my latest commit, also contains a blur fix. I didn't know which 'map'  you were talking about, but I assumed it was the main menu with the map. (Try to provide screenshot or more detailed information about where it is exactly next time :)) That one is now fixed, but feel free to redownload the latest packs and test them.

**Comment by daninthemix** [comment]
Just tested and the problem isn't fixed. The title screen is fine - you need to actually load a saved game (any saved game), then when the map comes up it's all black.

**Comment by Crementif** [comment]
I've just tested this and it doesn't seem to work indeed. Think this is the Cemu upscaling bug, so for the time being it's better to stay on native resolution. Check progress on this bug in this github issue here: https://github.com/slashiee/cemu_graphic_packs/issues/126 .

**Comment by Crementif** [comment]
Might not be related to Cemu, so opened again.

**Comment by Crementif** [comment]
Commit https://github.com/slashiee/cemu_graphic_packs/commit/d95c760c5c9bc0501bff28ab7425dfafe49132e0 fixed this issue. Due to Nsight changes recentely made in Nsight (seperation of cemu settings when ran), I thought I fixed it. I just discovered that but I'm glad I found why it worked 'that one time'. Anyhow, with some more research I found something hidden in the textures which didn't scale properly. Let me know if this commit has fixed all the issues you mentioned, closing the issue.
